Visual tweaks to some screens

pzapataf opened the pull request.

vblagoje commented:
I'll take this one @tristantarrant I'll collapse them all in one commit @pzapataf before I integrate them into master.

vblagoje commented:
@pzapataf looks good but I noticed that "Actions" pull down menus now require two clicks to get activated. I tried both Chrome and Firefox. Would you please confirm what I experienced and if so - please investigate why is that the case! On the current master, I can not notice this issue!

vblagoje commented:
@pzapataf I found the culprit - it is dropdown.js introduced in these commits When I removed it I got the the desired menu drop-down behaviour. Do we really need dropdown.js?

vblagoje commented on src/main/webapp/cluster-view/cluster-view.html:
```diff
@@ -17,11 +17,6 @@
-          <li role="presentation" class="divider"></li>
-          <li role="presentation" class="disabled"><a id="dropdownMenuDelete" role="menuitem">Delete</a>
-          <li role="presentation" class="divider"></li>
           <li role="presentation" ng-class="currentCluster.rebalancingStatus ? 'disabled' : 'enabled'"><a  id="dropdownMenuEnableRebalance"  role="menuitem" ng-click="confirmAndSetCacheContainerRebalance(true, 'ENABLE rebalancing for cache container?');">Enable rebalancing</a>
```
I believe the HTML structure we want to here is 
````
<li><a></a></li>
`````

vblagoje commented on src/main/webapp/cluster-view/tasks-view.html:
```diff
@@ -16,24 +16,53 @@
-        <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu1">
-          <li role="presentation" class="disabled"><a role="menuitem">Start</a></li>
-          <li role="presentation" class="disabled"><a role="menuitem">Stop</a></li>
+        <ul class="dropdown-menu" role="menu" aria-labelledby="dropdownMenu">
```
Here as well:
````
<li><a></a></li>
`````

pzapataf commented:
Fixed the issue, removed the dropdown.js dependency and changed the style of the logout menu. However, a small issue persists when the resolution is collapsed to minimum size, that menu is not expanded on click.

vblagoje commented:
I think that is not a big issue at this time @pzapataf We can tackle it later. I am going to squash these commits now and integrate it to master!

vblagoje commented:
Integrated with 6de28b371e2632ff56f2150f146e5757a9d98a3d Thanks @pzapataf
